data/deep missions.txt

```diff
@@ -40,24 +40,29 @@ mission "Deep: Syndicate Convoy"
 	to offer
 		"mystery cube" >= 10
 		or
-			random < 30
+			random < 20
 			and
 				"deep convoy" == 1
-				random < 40
+				random < 30
 			and
 				"deep convoy" == 2
-				random < 50
+				random < 40
 		
 	on offer
 		conversation
-			`You are approached by a man in a Deep Security uniform who asks you if you would be willing to escort a few freighters to <stopovers> and back by <date>. "We would normally have DS ships do it," he says, "but we're short on available ships given certain circumstances."`
+			`You are approached by a man in a Deep Security uniform who asks you if you would be willing to escort a few freighters to <stopovers> and back by <date>. "We would normally have our own fleet do it," he says, "but we're short on available ships given certain circumstances."`
 			choice
 				`	"Sure, I can do that."`
 				`	"Sorry, but I will not be traveling in that direction any time soon."`
-					decline
+					defer
 			
-			`	"Good. Before you leave, know that there may be pirate forces waiting for you in Tania Australis, Merak, and Altair. You can try to avoid them, but make sure you get the freighters back on time."`
-			`	The Deep Security officer wishes you good luck and walks off, leaving you wonder how he knows where pirate fleets might be waiting for you.`
+			branch warship
```

Comment:
Nonononononono

The point of the DS officers mentioning the systems regardless of what the player has is to hint at Deep: Mystery Cubes, which will reveal the cubes as being sensor devices (see Deep: Remnant Surveillance). Also, this branch doesn't make sense if the player has two warships, as the very last thing stated assumes that you were told about certain systems, but those systems are never mentioned if you have two warships.